lldb/source/Plugins/SymbolFile/DWARF/SymbolFileDWARFDebugMap.cpp

```diff
@@ -1565,10 +1567,12 @@ SymbolFileDWARFDebugMap::GetASTData(lldb::LanguageType language) {
                         __FUNCTION__, file_spec.GetPath().c_str());
         }
       } else {
-        if (log)
-          log->Printf("SymbolFileDWARFDebugMap::%s() - found reference to AST "
-                      "file %s, but could not find the file, ignoring",
-                      __FUNCTION__, file_spec.GetPath().c_str());
+        for (Log *current_log : {log, GetLog(LLDBLog::Types)})
+          if (current_log)
+            current_log->Printf("SymbolFileDWARFDebugMap::%s() - ignoring "
+                                "nonexistent AST file %s - referenced from %s",
+                                __FUNCTION__, file_spec.GetPath().c_str(),
+                                m_objfile_sp->GetFileSpec().GetPath().c_str());
```

Comment:
might be a good opportunity to convert this to LLDB_LOG (no F) and get rid of the c_str() conversions...